dev_guide/builds.adoc

```diff
@@ -549,189 +615,489 @@ source:
 
 // end::using-a-proxy-for-git-cloning-1[]
 
-[[starting-a-build]]
 
-== Starting a Build
-Manually invoke a build using the following command:
+[#using-private-repositories-for-builds]
+=== Using Private Repositories for Builds
 
-----
-$ oc start-build <BuildConfigName>
-----
+Supply valid credentials to build an application from a private repository.
 
-Re-run a build using the `--from-build` flag:
+Currently two types of authentication are supported: basic username-password
+and SSH key based authentication.
```

Comment:
Again, this is in the original docs... maybe we want to fix this one in a separate PR?